### PR TITLE
Update policy-csp-localpoliciessecurityoptions.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2189

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md [hunk 1]
```diff
@@ -2638,6 +2638,9 @@ GP Info:
 <!--Policy-->
 <a href="" id="localpoliciessecurityoptions-recoveryconsole-allowautomaticadministrativelogon"></a>**LocalPoliciesSecurityOptions/RecoveryConsole_AllowAutomaticAdministrativeLogon**  
 
+> [!Warning]  
+> Starting in the version 1803 of Windows, this policy is deprecated.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md]:
```suggestion
> Starting with Windows 10 version 1803, this policy is deprecated.
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were successfully applied
Thank you!

**Pull request merged**